### ISSUE TITLE
Quasar crashing at start-up on KODI 15.2

### ISSUE THREAD

**Issue by RLpolo77** [scakemyer/plugin.video.quasar#506]
Have used Quasar for a week and now suddenly it won't start-up any more.
Running on Openelec 6.0.3 (KODI 15.2)

See part of the kodi.log below.
```
01:00:19  19.334204 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m
01:00:19  19.335024 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m________
01:00:19  19.343130 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m\_____  \  __ _______    ___________ _______
01:00:19  19.349951 T:1777857600  NOTICE: Universal - An Addons Toolkit: -watchhistory- -Loading sqlite3 as DB engine version: 3.11.0
01:00:19  19.350353 T:1777857600  NOTICE: Universal - An Addons Toolkit: - watchhistory - -Auto Cleanup Start
01:00:19  19.351618 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m /  / \  \|  |  \__  \  /  ___/\__  \\_  __ \
01:00:19  19.374290 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m/   \_/.  \  |  // __ \_\___ \  / __ \|  | \/
01:00:19  19.379642 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m\_____\ \_/____/(____  /____  >(____  /__|
01:00:19  19.380615 T:1777857600  NOTICE: Universal - An Addons Toolkit: -watchhistory- -DELETE FROM watch_history WHERE lastwatched < '1969-12-12'
01:00:19  19.381964 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m       \__>          \/     \/      \/
01:00:19  19.387936 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m
01:00:19  19.391684 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0mVersion: "v0.9.34" Go: go1.6.2
01:00:19  19.392000 T:1777857600  NOTICE: Universal - An Addons Toolkit: - watchhistory - -Auto Cleanup End
01:00:19  19.402098 T:1523577920  NOTICE: [plugin.video.quasar] INFO  config       ▶ Reload           [0mReloading configuration...
01:00:19  19.477722 T:1531966528  NOTICE: Thread FileCache start, auto delete: false
01:00:19  19.858419 T:1710744640  NOTICE: Previous line repeats 2 times.
01:00:19  19.858576 T:1710744640  NOTICE: 0 Stale Textures Removed
01:00:20  20.343304 T:1777857600  NOTICE: Thread FileCache start, auto delete: false
01:00:20  20.683052 T:1654649920   ERROR: CCurlFile::FillBuffer - Failed: HTTP returned error 403
01:00:20  20.683203 T:1654649920   ERROR: CCurlFile::Open failed with code 403 for http://www.sosrepo.tv/repo/addons.xml.md5
01:00:20  20.683308 T:1654649920   ERROR: Open - failed to open source <http://www.sosrepo.tv/repo/addons.xml.md5>
01:00:20  20.683439 T:1654649920   ERROR: Failed to fetch checksum for directory listing http://www.sosrepo.tv/repo/addons.xml for repository repository.sos.
01:00:20  20.732048 T:1523577920  NOTICE: [plugin.video.quasar] runtime: no module data for 0x140b4365
01:00:20  20.732599 T:1523577920  NOTICE: [plugin.video.quasar] fatal error: no module data
01:00:20  20.733835 T:1523577920  NOTICE: [plugin.video.quasar]
01:00:20  20.734213 T:1523577920  NOTICE: [plugin.video.quasar] runtime stack:
01:00:20  20.734840 T:1523577920  NOTICE: [plugin.video.quasar] runtime.throw(0xa35038, 0xe)
01:00:20  20.735416 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/panic.go:547 +0x78
01:00:20  20.736078 T:1523577920  NOTICE: [plugin.video.quasar] runtime.pcvalue(0xc18214, 0x18423bf8, 0x2b4950, 0x446fead0, 0x123701, 0xad3d10)
01:00:20  20.736700 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/symtab.go:273 +0x178
01:00:20  20.737282 T:1523577920  NOTICE: [plugin.video.quasar] runtime.funcspdelta(0xc18214, 0x2b4950, 0x446fead0, 0x18)
01:00:20  20.737799 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/symtab.go:363 +0x40
01:00:20  20.738384 T:1523577920  NOTICE: [plugin.video.quasar] runtime.gentraceback(0x1255e0, 0x56c42ab0, 0x1255b0, 0x56de21a0, 0x0, 0x0, 0x7fffffff, 0xb0b384, 0x446fec3c, 0x0, ...)
01:00:20  20.738853 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/traceback.go:255 +0x488
01:00:20  20.739449 T:1523577920  NOTICE: [plugin.video.quasar] runtime.copystack(0x56de21a0, 0x1000)
01:00:20  20.739964 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/stack.go:759 +0x150
01:00:20  20.740480 T:1523577920  NOTICE: [plugin.video.quasar] runtime.newstack()
01:00:20  20.741060 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/stack.go:947 +0xae8
01:00:20  20.745461 T:1523577920  NOTICE: [plugin.video.quasar] System information: linux_armv7 4.1.18
01:00:20  20.745796 T:1523577920  NOTICE: [plugin.video.quasar] Kodi build version: 15.2 Git:02e7013
01:00:20  20.746059 T:1523577920  NOTICE: [plugin.video.quasar] OS type: Linux
01:00:20  20.746342 T:1523577920  NOTICE: [plugin.video.quasar] uname: ('Linux', 'SOSBOX', '4.1.18', '#1 SMP Mon Feb 29 20:48:22 CET 2016', 'armv7l', '')
01:00:20  20.777609 T:1427108928  NOTICE: Thread FileCache start, auto delete: false
01:00:23  23.761831 T:1523577920  NOTICE: Previous line repeats 6 times.
01:00:23  23.761940 T:1523577920  NOTICE: [plugin.video.quasar] quasard: starting quasard
01:00:23  23.765175 T:1523577920  NOTICE: [plugin.video.quasar] Deleting stale files set([])
01:00:24  24.061443 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m
01:00:24  24.062592 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m________
01:00:24  24.067467 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m\_____  \  __ _______    ___________ _______
01:00:24  24.068718 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m /  / \  \|  |  \__  \  /  ___/\__  \\_  __ \
01:00:24  24.072834 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m/   \_/.  \  |  // __ \_\___ \  / __ \|  | \/
01:00:24  24.073652 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m\_____\ \_/____/(____  /____  >(____  /__|
01:00:24  24.074606 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m       \__>          \/     \/      \/
01:00:24  24.079765 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m
01:00:24  24.081255 T:1523577920  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0mVersion: "v0.9.34" Go: go1.6.2
01:00:24  24.085939 T:1523577920  NOTICE: [plugin.video.quasar] INFO  config       ▶ Reload           [0mReloading configuration...
01:00:24  24.248316 T:1654649920   ERROR: CCurlFile::FillBuffer - Failed: Couldn't resolve host name(6)
01:00:24  24.248449 T:1654649920   ERROR: CCurlFile::Open failed with code 0 for http://kodi2.speedbox.me/svn_kodi/trunk/addons.xml.md5
01:00:24  24.248549 T:1654649920   ERROR: Open - failed to open source <http://kodi2.speedbox.me/svn_kodi/trunk/addons.xml.md5>
01:00:24  24.248659 T:1654649920   ERROR: Failed to fetch checksum for directory listing http://kodi2.speedbox.me/svn_kodi/trunk/addons.xml for repository repository.kodiunderground.
01:00:24  24.315037 T:1777857600  NOTICE: Thread FileCache start, auto delete: false
01:00:24  24.446869 T:1523577920  NOTICE: [plugin.video.quasar] runtime: no module data for 0x140b4365
01:00:24  24.447367 T:1523577920  NOTICE: [plugin.video.quasar] fatal error: no module data
01:00:24  24.448730 T:1523577920  NOTICE: [plugin.video.quasar]
01:00:24  24.449121 T:1523577920  NOTICE: [plugin.video.quasar] runtime stack:
01:00:24  24.449451 T:1523577920  NOTICE: [plugin.video.quasar] runtime.throw(0xa35038, 0xe)
01:00:24  24.449823 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/panic.go:547 +0x78
01:00:24  24.450567 T:1523577920  NOTICE: [plugin.video.quasar] runtime.pcvalue(0xc18214, 0x18423bf8, 0x2b4950, 0x450fead0, 0x123701, 0xad3d10)
01:00:24  24.451008 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/symtab.go:273 +0x178
01:00:24  24.451385 T:1523577920  NOTICE: [plugin.video.quasar] runtime.funcspdelta(0xc18214, 0x2b4950, 0x450fead0, 0x18)
01:00:24  24.451742 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/symtab.go:363 +0x40
01:00:24  24.452187 T:1523577920  NOTICE: [plugin.video.quasar] runtime.gentraceback(0x1255e0, 0x56dd92b0, 0x1255b0, 0x56c5e9c0, 0x0, 0x0, 0x7fffffff, 0xb0b384, 0x450fec3c, 0x0, ...)
01:00:24  24.452606 T:1523577920  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/traceback.go:255 +0x488
01:00:24  24.453108 T:1523577920  NOTICE: [plugin.video.quasar] System information: linux_armv7 4.1.18
01:00:24  24.453379 T:1523577920  NOTICE: [plugin.video.quasar] Kodi build version: 15.2 Git:02e7013
01:00:24  24.453646 T:1523577920  NOTICE: [plugin.video.quasar] OS type: Linux
01:00:24  24.453924 T:1523577920  NOTICE: [plugin.video.quasar] uname: ('Linux', 'SOSBOX', '4.1.18', '#1 SMP Mon Feb 29 20:48:22 CET 2016', 'armv7l', '')
01:00:24  24.662344 T:1777857600  NOTICE: Thread FileCache start, auto delete: false
01:00:26  26.315071 T:1654649920  NOTICE: Previous line repeats 3 times.
```

Thanks for getting this resolved!

**Comment by RLpolo77** [comment]
Update: Reverted back to version 0.9.33 which resolved the problem on OpenElec 6.0.3 (KODI 15.2)
Could this be related to the upgrade of GO lang in 0.9.34?

**Comment by bluenote73** [comment]
Would you mind trying a latest (I suggest libre elec) fresh build and see if this continues to occur?
thanks
